lib/models/Review.js

```diff
@@ -0,0 +1,46 @@
+const pool = require('../utils/pool');
+
+module.exports = class Review {
+  id;
+  user_id;
+  restaurant_id;
+  stars;
+  detail;
+
+  constructor(row) {
+    this.id = row.id;
+    this.user_id = row.user_id;
+    this.restaurant_id = row.restaurant_id;
+    this.stars = row.stars;
+    this.detail = row.detail;
+  }
+  // GET BY ID FUNCTION
+  static async getReviewId(id) {
+    const { rows } = await pool.query(
+      `
+      SELECT * FROM reviews WHERE id = $1`,
+      [id]
+    );
+    if (!rows[0]) return null;
+    return new Review(rows[0]);
+  }
+  // DELETE FUCNTION
+  static async delete(id) {
+    const { rows } = await pool.query(
+      `
+    DELETE FROM reviews 
+    WHERE id = $1
+    RETURNING *`,
+      [id]
+    );
```

Comment:
you'll want some kind of check here to handle if there is no review that matches the id — because if there isn't ```rows``` will be ```undefined``` and then when you try to make a ```new Review(rows[0])``` it'll break, something like ```if (!rows[0]) return null```